src/rpp/rpp/sources/interval.hpp

```diff
@@ -0,0 +1,97 @@
+//                   ReactivePlusPlus library
+//
+//           Copyright Aleksey Loginov 2023 - present.
+//  Distributed under the Boost Software License, Version 1.0.
+//     (See accompanying file LICENSE_1_0.txt or copy at
+//           https://www.boost.org/LICENSE_1_0.txt)
+//
+//  Project home: https://github.com/victimsnino/ReactivePlusPlus
+
+#pragma once
+
+#include <rpp/sources/fwd.hpp>
+#include <rpp/observables/observable.hpp>
+
+namespace rpp::details
+{
+struct interval_schedulable
+{
+    rpp::schedulers::optional_duration operator()(auto&& observer, rpp::schedulers::duration period, size_t& counter) const
```

Comment:
i think, you can get observer like const auto&

Reply:
done